components/layout/LoginBanner.js

```diff
@@ -48,7 +47,10 @@ const BFBanner = () => {
 
   React.useEffect(() => {
     const hideBanner = localStorage.getItem('hideLoginBanner');
-    if (hideBanner === 'true') {
+    if (showBanner) {
```

Comment:
showBanner and hideBanner looks like the same variable but inverted `hideBanner` is equal to `!showBanner`?